pkgs/top-level/all-packages.nix

```diff
@@ -21435,6 +21436,20 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) CoreFoundation Security;
   };
 
+  mongodb-4_4 = callPackage ../servers/nosql/mongodb/v4_4.nix {
+    sasl = cyrus_sasl;
+    boost = boost170.override { enableShared = false; };
+    inherit (darwin) cctools;
+    inherit (darwin.apple_sdk.frameworks) CoreFoundation Security;
+  };
+
+  mongodb-5_0 = callPackage ../servers/nosql/mongodb/v5_0.nix {
+    sasl = cyrus_sasl;
+    boost = boost176.override { enableShared = false; };
```

Comment:
Is this still necessary? 1.77 is default in master now and per your previous comment mongo is apparently building with 1.77.

Reply:
Just mentioned this above. Previous versions seem to have always used the minimum required boost, so I wanted to be consistent with that. Happy to change that, but I think we have to keep the override to force static linking.

Reply:
I'm of the opinion you should always point to the most recent boost library the program works with, not sure why you would want to use the oldest possible.

Reply:
No worries. I'll bump to 1.77

Reply:
```suggestion
    boost = boost17x.override { enableShared = false; };
```